docs/tutorials/exptime.rst

```diff
@@ -0,0 +1,492 @@
+.. _exptime_tutorial:
+
+.. doctest-skip-all
+
+****************************
+Predicting an Exposure Time
+****************************
+
+Contents
+========
+
+ In this tutorial we will show how to predict the exposure time needed
+ to obtain a given signal to noise ratio for a target star.
+
+* :ref:`exptime-querying_target_properties`
+* :ref:`exptime-obtaining_a_model_spectrum`
+* :ref:`exptime-setting_the_location_of_the_observer`
+* :ref:`exptime-creating_a_telescope_model`
+* :ref:`exptime-getting_the_exposure_time`
+* :ref:`exptime-adding_an_atmospheric_transmission_model`
+
```

Comment:
Perhaps you could add a note up here, at the very top, which says something like: "This module of astroplan requires scipy and synphot." with links to the scipy and synphot installation documentation. That way no one is surprised later on in the tutorial.